bahamas/app/models.py

```diff
@@ -0,0 +1,22 @@
+from bahamas.app import db
+
+
+class Client(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    fiscal_id = db.Column(db.String(20), unique=True)
+    email = db.Column(db.String(120), unique=True)
+    name = db.Column(db.String(64))
+    invoices = db.relationship('Invoice', backref='rel_sec_client', lazy='dynamic')
+
+    def __repr__(self):
+        return '<User {}>'.format(self.email)
+
+
+class Invoice(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
Every time I created a database I always had an ID, this time I could have used the invoice_id easily but one thing that I wanted, is to have every type as a string, this way I can work every field the same way and transform the data type as I need it.

A chose an SQL database, in this case, an SQLite database because it's the main I have used in my few years of programming, and because I felt like it was the easiest to work within this app.